WordPressVIPMinimum/ruleset-test.php

```diff
@@ -262,6 +247,21 @@
 		288 => 1,
 		293 => 1,
 		294 => 1,
+		368 => 1,
```

Comment:
Should the changes in this file be reverted? The change to Warnings should now only be for `WordPress-VIP-Go`, not `WordPressVIPMinimum`.

Reply:
Yes. Definitely will need to revert those. I'll work on that.